packages/lib-classifier/src/plugins/tasks/Registry.js

```diff
@@ -0,0 +1,29 @@
+class Registry {
+  constructor () {
+    this.register = {}
+  }
+
+  get (key) {
+    return this.register[key] || {}
```

Comment:
what's the benefit to returning an empty object here?

Reply:
`registry.get(type).TaskModel` will be undefined rather than crashing the classifier.